### PR TITLE
CODE RUB: Rename Brokers/API to Brokers/Apis along with the namespace

### PR DESCRIPTION
**Pull request by rcruzfreelance**

### CODE REVIEW

**Comment by rcruzfreelance** [comment]
Closes #[12](https://github.com/hassanhabib/Taarafo.Web/issues/12)

File: OtripleS.Portal.Web/Brokers/Apis/IApiBroker.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+namespace OtripleS.Portal.Web.Brokers.Apis
+{
+    public partial interface IApiBroker
+    {
+
```
Comment (ShriHumrudha) [inline OtripleS.Portal.Web/Brokers/Apis/IApiBroker.cs:10]: We can get rid of this space

**Comment by ShriHumrudha** [comment]
> Closes #[12](https://github.com/hassanhabib/Taarafo.Web/issues/12)

Did you build the solution? I wonder we need to update all other references with the same namespace too.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: OtripleS.Portal.Web/Brokers/Apis/IApiBroker.cs (hunk 1 above)
Comment (rcruzfreelance) [reply OtripleS.Portal.Web/Brokers/Apis/IApiBroker.cs]: I was able to do the build without no problem

image

Comment (ShriHumrudha) [reply OtripleS.Portal.Web/Brokers/Apis/IApiBroker.cs]: Interesting. I was referring to these where the namespace still needs 'Apis'

image

Comment (rcruzfreelance) [reply OtripleS.Portal.Web/Brokers/Apis/IApiBroker.cs]: Thank you for the clarification. I made the changes

**Comment by hassanhabib** [comment]
@rcruzfreelance please fix the conflicts

**Comment by rcruzfreelance** [comment]
I see that I don't have write access to resolve the conflicts

**Comment by ShriHumrudha** [comment]
> @hassanhabib I see that I don't have write access to resolve the conflicts. Can you help me out with this?

You can do it in your IDE. Take the latest of master into your user branch, it will list the conflicts to perform the action.

**Comment by rcruzfreelance** [comment]
I was able to pull from master to my user branch but im not getting any conflict messages

image

**Comment by rcruzfreelance** [comment]
image

**Comment by ShriHumrudha** [comment]
> I was able to pull from master to my user branch but im not getting any conflict messages
> 
> image

Can you confirm if your forked repository has the latest?

Steps for resolving:
Update your forked repo
Pull latest in your IDE
Merge master into this user branch
This should list the differences.

**Comment by rcruzfreelance** [comment]
Yes I did the exact same process and had no conflicts in my IDE

**Comment by ShriHumrudha** [comment]
Step 1: Navigate to your forked instance. It says, 41 commits are behind the latest.

image

Step 2: Get the latest

image

**Comment by rcruzfreelance** [comment]
ok i did not remember the fetch upstream in github, thank you @ShriHumrudha . I was able to merge the conflict to my forked repository. I should commits ahead now

image

**Comment by ShriHumrudha** [comment]
> ok i did not remember the fetch upstream in github, thank you @ShriHumrudha . I was able to merge the conflict to my forked repository. I should commits ahead now
> 
> image

Please fix the build errors.

image

**Pull request merged**